Lib/idlelib/pyshell.py

```diff
@@ -387,6 +387,16 @@ def handle_EOF(self):
         "Override the base class - just re-raise EOFError"
         raise EOFError
 
+def restart_line(width, filename):  # See bpo-38141.
```

Comment:
I know you like short names, but I find this unclear. I suggest "format_restart_line". Otherwise, let's add a doc-string.

Reply:
Also, this might better be a classmethod or staticmethod rather than a module-level function.

Reply:
Docstrings for function are a given. I will add one with the specification being implemented.